### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by jeffjennings**
This is the last PR needed before a new version of the code (v.1.1.0) is pushed to PyPI. It's a good time to push a new version because we've done a number of bug fixes since the v.1.0.0 release, a couple of which would warrant an update on their own. 

The PR: 
- mainly updates the figures and a bit of text in the docs to be consistent with the current version of the code,
- makes a couple small changes in `make_figs` to improve figures,
- changes the code version in `__init__` to v1.1.0 (from 1.0.0).

Once this PR is merged, I'll tag master (on github) as a new release, v1.1.0, and push master to PyPI.

### CODE REVIEW

File: docs/tutorials/fit_convergence.rst [hunk 1]
```diff
@@ -17,26 +17,25 @@ of AS 209 from the Quickstart, looks like this,
...
+This criterion using the power spectrum is more robust than one based on the brightness profile because of the oscillations imposed on the latter by sparse sampling.
+If this stopping condition is *not* met, the fit runs until `max_iter` as set in your parameter file. |br|
 **b)** Sequential difference between the last 100 brightness profile iterations.
-Note the y-scale here is in :math:`10^5\ {\rm Jy\ sr}^{-1}` (compared to :math:`10^{10}\ {\rm Jy\ sr}^{-1}` in (a)),
-so in this case the oscillations remaining at the end of the fit (:math:`\approx 1250` iterations) are at parts in :math:`10^6`.
+The oscillations remaining at the end of the fit (:math:`\approx 630` iterations) are at a couple parts in :math:`10^5`.
 |br|
 **c)** The reconstructed power spectrum over all fit iterations.
-Our initial guess for the power spectrum, a power law with slope of -2, is apparent in the longest baselines for the first :math:`\approx 250` iterations,
+Our initial guess for the power spectrum, a power law with slope of -2, is apparent in the longest baselines for the first :math:`\approx 100` iterations,
```
Comment (rbooth200) [inline docs/tutorials/fit_convergence.rst:31]: I think you should point out that the number of iterations is \alpha dependent

**Review by rbooth200 — COMMENTED** [review]
So I wasn't aware we were doing a v1.1.0 a this time... It's worth getting @mtazzari to discuss what else we should include in a new release. A changelog maybe?

File: docs/tutorials/fit_convergence.rst (hunk 1 above)
Comment (jeffjennings) [reply docs/tutorials/fit_convergence.rst]: Good point, added in #700d119

**Comment by jeffjennings** [comment]
@rbooth200, I've made one extra change since you reviewed, in 6cbb37b.

**Comment by jeffjennings** [comment]
Hey @mtazzari, would you mind reviewing Richard's question in this PR so we can merge and close this? I'm happy to make any changes that come to mind of course. Thanks!

File: frank/make_figs.py [hunk 2]
```diff
@@ -528,14 +527,13 @@ def make_full_fig(u, v, vis, weights, sol, bin_widths, alpha, wsmooth,
 
         # Plot a sweep over 2\pi of the frank 1D fit
         # (analogous to a model image of the source)
+        vmax =sol.mean.max()
```
Comment (mtazzari) [inline frank/make_figs.py:530]:
```suggestion
        vmax = sol.mean.max()
```

File: frank/make_figs.py [hunk 3]
```diff
@@ -700,10 +698,10 @@ def make_diag_fig(r, q, iteration_diagnostics, iter_plot_range=None,
...
 
         # Plot the difference in the power spectrum between the last 100 iterations
-        plot_pwr_spec_iterations(q, np.diff(pwr_spec_iter, axis=0),
-                                 iter_plot_range_end, ax3,
-                                 cmap=plt.cm.cividis,  # pylint: disable=no-member
-                                 bbox_x=.45)
+        plot_pwr_spec_iterations(q, abs(np.diff(pwr_spec_iter, axis=0)),
+                                iter_plot_range_end, ax3,
+                                cmap=plt.cm.cividis,  # pylint: disable=no-member
+                                bbox_x=.45)
```
Comment (mtazzari) [inline frank/make_figs.py:704]:
```suggestion
                                 iter_plot_range_end, ax3,
                                 cmap=plt.cm.cividis,  # pylint: disable=no-member
                                 bbox_x=.45)
```

Comment (mtazzari) [reply frank/make_figs.py:704]: just reverting to the correct alignment

File: HISTORY.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    - Adds a couple logging messages
+- Docs:
+    - Updates jupyter notebooks and figures to be consistent with current code version
+    - Updates paper links to point to ADS
+    - Adds changelog ('HISTORY.rst')
+    - Adds video tutorial
+    - Adds more verbose descriptions in some parts of docs
+- Miscellaneous:
+    - Fixes 'MaskedArray' warnings in 'utilities.py', 'plot.py'
+    - Fixes a few other minor bugs
```
Comment (mtazzari) [inline HISTORY.rst:65]: Typically the order is such that the most recent release is first.
Please consider reversing the order here.

**Review by mtazzari — CHANGES_REQUESTED** [review]
Great work!
I did not test the code quantitatively so I trust you and Richard on this.
Totally fine with naming this v1.1.0.
The changelog is a great idea! 
1) check that the changelog is linked in the docs
2) I've left some minor comments
3) the PR page says it is not up to date with master. I suggest you check out my review, do the changes, and as a very last thing you push "update branch" on the PR page. 
After this I'll be happy to approve.

File: HISTORY.rst (hunk 4 above)
Comment (jeffjennings) [reply HISTORY.rst]: done in 6a3549b

**Comment by jeffjennings** [comment]
> Great work!
> I did not test the code quantitatively so I trust you and Richard on this.
> Totally fine with naming this v1.1.0.
> The changelog is a great idea!
> 
> 1. check that the changelog is linked in the docs

Good point, the changelog is now linked in the README and on the docs site (commit 7a508fd).

> 2. I've left some minor comments

Thanks! I've integrated your changes (commits 0fbeb64, 5316100, 6a3549b). 

> 3. the PR page says it is not up to date with master. I suggest you check out my review, do the changes, and as a very last thing you push "update branch" on the PR page.
>    After this I'll be happy to approve.

Done!

**Review by mtazzari — APPROVED** [review]

**Pull request merged**